admin/services/config.service.js

```diff
@@ -38,7 +41,29 @@ const configService = {
 
     // specific config for a pupil
     const checkOptions = {
-      speechSynthesis: !!pupil.speechSynthesis
+      speechSynthesis: !!pupil.speechSynthesis,
+      audibleSounds: false
+    }
+
+    let accessArrangements
+    let pupilAccessArrangements
+    try {
+      accessArrangements = await accessArrangementsService.getAccessArrangements()
+      pupilAccessArrangements = await pupilAccessArrangementsDataService.sqlFindAccessArrangementsByPupilId(pupil.id)
```

Comment:
Please bear in mind that at some point this service will need to upgrade to be a batch service for many pupils at once.

Reply:
Indeed - we could eventually update to accept multiple pupils once the Azure queue work is complete 👍